Readme.md

```diff
@@ -0,0 +1,52 @@
+qqsort
```

Comment:
Needs a license file.  Default for this repo is currently proprietary, which we can't use.